libs/methodologies/bold/rule-processors/mass-id/avoided-emissions/src/avoided-emissions.constants.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  MassIdOrganicSubtype,
+  MethodologyBaseline,
+} from '@carrot-fndn/shared/methodologies/bold/types';
+
+export const AVOIDED_EMISSIONS_BASELINE_MATRIX: Record<
+  MassIdOrganicSubtype,
+  Record<MethodologyBaseline, number>
+> = {
+  [MassIdOrganicSubtype.DOMESTIC_SLUDGE]: {
+    [MethodologyBaseline.LANDFILLS_WITH_FLARING_OF_METHANE_GAS]: 0.067,
+    [MethodologyBaseline.LANDFILLS_WITHOUT_FLARING_OF_METHANE_GAS]: 0.227,
+    [MethodologyBaseline.OPEN_AIR_DUMP]: 0.156,
+  },
+  [MassIdOrganicSubtype.EFB_SIMILAR_TO_GARDEN_YARD_AND_PARK_WASTE]: {
+    [MethodologyBaseline.LANDFILLS_WITH_FLARING_OF_METHANE_GAS]: 0.629,
+    [MethodologyBaseline.LANDFILLS_WITHOUT_FLARING_OF_METHANE_GAS]: 1.251,
+    [MethodologyBaseline.OPEN_AIR_DUMP]: 0.975,
+  },
+  [MassIdOrganicSubtype.FOOD_FOOD_WASTE_AND_BEVERAGES]: {
+    [MethodologyBaseline.LANDFILLS_WITH_FLARING_OF_METHANE_GAS]: 0.459,
+    [MethodologyBaseline.LANDFILLS_WITHOUT_FLARING_OF_METHANE_GAS]: 0.94,
+    [MethodologyBaseline.OPEN_AIR_DUMP]: 0.727,
```

Comment:
If we use the spreadsheet with 1 ton, these factor is 0.726, but if we use 1000 tons it is 0.727, so there is some small imprecision here. I think we should round down to be conservative. Another approach is to use more decimal cases.